backend/app/crud/__init__.py

```
@@ -64,3 +64,25 @@
     create_conversation,
     delete_conversation,
 )
+
+from .prompt import (
+    count_prompts_by_project,
```

Ruff (0.12.2)

`.prompt.count_prompts_by_project` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)